### PR TITLE
Add README and all messaging

### PR DESCRIPTION
**Pull request by johnbieren**
Please give this a decent lookover, as I would not be surprised if I made a typo somewhere. Specifically the Jenkinsfile and library file. The only messages I did not add were the package.queued and package.ignored ones, as they go in JenkinsfileTrigger. Can you add those? You have a better grasp on that file than me. 

One thing to think about is maybe we change the library for this workflow to allpackagespipelineutils instead of packagepipelineutils? Might make things more clear with the topic I picked and shouldn't be too hard...

Will need to make a PR upstream for the pull_old_task and build_local scripts in the rpmbuild container to define nvr and original_spec_nvr so it works with the pagure tooling. I will do that

Signed-off-by: Johnny Bieren <jbieren@redhat.com>

### CODE REVIEW

**Comment by johnbieren** [comment]
@pypingou As you can see here the topic for this all packages pipeline will be
org.centos.prod.allpackages.pipeline.complete
(and the corresponding stage ones will be on org.centos.prod.allpackages.pipeline.*)

**Comment by johnbieren** [comment]
Will need this PR for the nvr part of package messages
https://github.com/CentOS-PaaS-SIG/ci-pipeline/pull/537

File: Jenkinsfile [hunk 1]
```diff
@@ -189,19 +200,16 @@ podTemplate(name: podName,
...
-                    // out in case the future intent is to use it
-
                     // Set stage specific vars
                     packagepipelineUtils.handlePipelineStep('stepName': currentStage, 'debug': true) {
                         packagepipelineUtils.setStageEnvVars(currentStage)
 
 
                         // Return a map (messageFields) of our message topic, properties, and content
-                        //messageFields = pipelineUtils.setMessageFields("package.running")
+                        messageFields = packapgepipelineUtils.setMessageFields("package.running")
```
Comment (joejstuart) [inline Jenkinsfile]: Looks like typo in packagepipelineUtils

File: Jenkinsfile [hunk 2]
```diff
@@ -268,9 +290,9 @@ podTemplate(name: podName,
                         //packagepipelineUtils.setStageEnvVars(currentStage)
 
                         // Set our message topic, properties, and content
-                        //messageFields = pipelineUtils.setMessageFields("image.test.smoke.running")
+                        //messageFields = packagepipelineUtils.setMessageFields("image.test.smoke.running")
```
Comment (joejstuart) [inline Jenkinsfile]: Can the image-boot-sanity stage be deleted?

File: Jenkinsfile (hunk 1 above)
Comment (johnbieren) [reply Jenkinsfile]: Nice catch, thanks

File: Jenkinsfile (hunk 2 above)
Comment (johnbieren) [reply Jenkinsfile]: We could either delete it or we could make the check nvr part of it work

**Pull request merged**